lib/index.js

```diff
@@ -20,72 +20,66 @@ class AniList {
 	 *      the authorized user's profile (if set to private).
 	 * @param {InitOptions} [options] - Optional options to use when getting info from AniList
 	 */
-	constructor(accessKey, options = {}) {
-		if (!accessKey) {
-			accessKey = null;
-		}
-
+	constructor(accessKey = null, options = {}) {
 		if (options.timeout) {
 			if (typeof options.timeout !== "number") throw new TypeError("ERROR: 'options.timeout' should be a number");
-		} else {
-			options.timeout = 15000;
-		}
+		} else options.timeout = 15000;
```

Comment:
I'm going to flag this for code styling. Please keep the braces intact for if/else statements.